pkg/cmd/streamplace.go

```diff
@@ -87,11 +87,21 @@ func start(build *config.BuildFlags, platformJobs []jobFunc) error {
 	}
 
 	if len(os.Args) > 1 && os.Args[1] == "live" {
-		if len(os.Args) != 3 {
-			fmt.Println("usage: streamplace live [stream-key]")
+		cli := config.CLI{Build: build}
+		fs := cli.NewFlagSet("streamplace live")
+
+		err := cli.Parse(fs, os.Args[2:])
```

Comment:
This is fine but just a note to myself: I'm about to make `--public-host` a required parameter, will need to make sure that gets handled here